site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -80,7 +80,7 @@ spec:
       game: my-game
     matchExpressions:
       - {key: tier, operator: In, values: [cache]}
-    # [Stage:Alpha]
+    # [Stage:Beta]
```

Comment:
These changes will also need to be `feature` shortcoded, such that they only get updated when we do our next release.

You may want to make a copy of the entire example rather than try and do it within specific lines.

Reply:
I added `feature` shortcode like as you said.